packages/e2e-tests/specs/performance/post-editor.test.js

```diff
@@ -124,24 +160,25 @@ describe( 'Post Editor Performance', () => {
 
 		// Measuring typing performance
 		await insertBlock( 'Paragraph' );
-		i = 200;
+		i = 20;
```

Comment:
I noticed that the stability isn't improved a lot by using 200 or 20. So let's just make this run a bit quicker.